src/main/java/org/myrobotlab/service/InMoov.java

```diff
@@ -1761,30 +1761,21 @@ public void loadGestures() {
    * @param directory
    *          - the directory that contains the gesture python files.
    */
-  public void loadGestures(String directory) {
-    // TODO: iterate over each of the python files in the directory
+  public boolean loadGestures(String directory) {
+
+    // iterate over each of the python files in the directory
     // and load them into the python interpreter.
-    File dir = makeGesturesDirectory(directory);
+    File dir = new File(directory);
+    boolean loaded = true;
     for (File f : dir.listFiles()) {
```

Comment:
The gestures directory might not exist, you should be logging a message for that. and aborting the loading of scripts.